docs/quickstarts/packet.md

```diff
@@ -5,262 +5,243 @@
 * [Introduction](#introduction)
 * [Requirements](#requirements)
 * [Step 1: Install lokoctl](#step-1-install-lokoctl)
-* [Step 2: Set up a working directory](#step-2-set-up-a-working-directory)
-* [Step 3: Set up credentials from environment variables](#step-3-set-up-credentials-from-environment-variables)
-* [Step 4: Define cluster configuration](#step-4-define-cluster-configuration)
-* [Step 5: Create Lokomotive cluster](#step-5-create-lokomotive-cluster)
+* [Step 2: Create a cluster configuration](#step-2-create-a-cluster-configuration)
+* [Step 3: Deploy the cluster](#step-3-deploy-the-cluster)
 * [Verification](#verification)
 * [Cleanup](#cleanup)
 * [Troubleshooting](#troubleshooting)
-* [Conclusion](#conclusion)
 * [Next steps](#next-steps)
 
 ## Introduction
 
-This quickstart guide walks through the steps needed to create a Lokomotive cluster on Packet with
-Flatcar Container Linux using Route53 as the DNS provider.
+This guide shows how to create a Lokomotive cluster on [Packet](https://www.packet.com/). By the
+end of this guide, you'll have a basic Lokomotive cluster running on Packet.
 
-By the end of this guide, you'll have a production-ready Kubernetes cluster running on Packet.
+Lokomotive runs on top of [Flatcar Container Linux](https://www.flatcar-linux.org/). This guide
+uses the `stable` channel.
+
+The guide uses [Amazon Route 53](https://aws.amazon.com/route53/) as a DNS provider.
 
 ## Requirements
 
-* Basic understanding of Kubernetes concepts.
-* Packet account, Project ID and auth token (sometimes also referred to as [User Level API
-  key](https://www.packet.com/developers/docs/API/getting-started/)).
-* AWS account and IAM credentials (optional for Route53 DNS configuration).
-* AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
-* Terraform v0.12.x and [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct) v0.5.0
-  installed locally.
-* Local BGP enabled. More information on how to enable Local BGP for the Packet Project is found in
-  the [Packet support document](https://support.packet.com/kb/articles/bgp).
-* An SSH key pair for management access.
-* `kubectl` installed locally to access the Kubernetes cluster.
+* A Packet account with
+  [local BGP](https://www.packet.com/developers/docs/network/advanced/local-and-global-bgp/)
+  enabled.
+* A Packet project ID.
+* A Packet
+  [user level API key](https://www.packet.com/developers/docs/API/getting-started/)
+  with access to the relevant project.
+* An AWS account.
+* An AWS
+  [access key ID and secret](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+* An AWS Route 53 zone (can be a subdomain).
+* An SSH key pair for accessing the cluster nodes.
+* Terraform `v0.12.x` installed.
```

Comment:
Canonical link to install link for terraform: https://learn.hashicorp.com/terraform/getting-started/install.html#install-terraform